plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -100,4 +100,207 @@
             id="com.google.cloud.tools.eclipse.gcloud.runtime">
       </serverdefinition>
    </extension>
+   
+   <extension
+         point="org.eclipse.wst.server.core.runtimeTypes">
+      <runtimeType
+            class="com.google.cloud.tools.eclipse.appengine.localserver.runtime.CloudSdkRuntime"
+            description="%runtimeTypeDescription"
+            id="com.google.cloud.tools.eclipse.runtime.id1"
+            name="%runtimeTypeName"
+            vendor="%runtimeTypeVendor"
+            version="1">
+            <moduleType types="jst.web" versions="2.2, 2.3, 2.4, 2.5, 3.1"/>
+            <moduleType types="jst.connector" versions="1.0, 1.5, 1.6"/>
+            <moduleType types="jst.utility"  versions="1.0"/>
+      </runtimeType>
+   </extension>
+   <extension
+         point="org.eclipse.wst.server.ui.wizardFragments">
+      <fragment
+            class="com.google.cloud.tools.eclipse.appengine.localserver.ui.CloudSdkRuntimeWizardFragment"
+            id="com.google.cloud.tools.eclipse.runtime.fragment1"
+            typeIds="com.google.cloud.tools.eclipse.runtime.id1">
+      </fragment>
+   </extension>
+   <extension point="org.eclipse.wst.common.project.facet.core.facets">
+      <project-facet id="com.google.cloud.tools.eclipse.facet">
```

Comment:
I'm not sure what you mean by thumbs up. That's usually an LGTM from a reviewer.